k2/csrc/fsa_algo.cu

```diff
@@ -453,9 +453,130 @@ FsaVec LinearFsas(const Ragged<int32_t> &symbols) {
       arcs);
 }
 
+FsaVec LevenshteinGraphs(const Ragged<int32_t> &symbols,
+                         float penalty /* = -1 */,
```

Comment:
I think ins_del_score is a good name, since it explains what it is.
I'm surprised the default is -1 though, instead of -0.5.  I thought we were going to multiply all scores by 0.5, so that the total cost equals the Levenshtein alignment score.